sdk/iot/Azure.Iot.Hub.Service/samples/IotHubClientSamples/CommandLineOptions.cs

```diff
@@ -1,13 +1,20 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
+using System;
 using CommandLine;
 
 namespace Azure.Iot.Hub.Service.Samples
 {
     public class CommandLineOptions
     {
-        [Option('c', "iotHubConnectionString", Required = true, HelpText = "Iot Hub connection string")]
-        public string IotHubConnectionString { get; set; }
+        [Option('p', "sharedAccessPolicy", Required = true, HelpText = "Iot Hub instance shared access policy")]
```

Comment:
you can still use the connection string option by defining `SetName` in the Option constructor. Like so:

```
[Option('c', "iotHubConnectionString", Required = true, SetName = "ConnectionString", HelpText = "Iot Hub connection string")]
public string IotHubConnectionString { get; set; }

[Option('p', "sharedAccessPolicy", Required = true, SetName = "AccessPolicy", HelpText = "Iot Hub instance shared access policy")]
 public string IotHubSharedAccessPolicy { get; set; }


[Option('k', "sharedAccessKey", Required = true, SetName = "AccessPolicy", HelpText = "Iot Hub instance shared access key")]
public string IotHubSharedAccessKey { get; set; }

[Option('e', "endpoint", Required = true, SetName = "AccessPolicy", HelpText = "Iot Hub endpoint")]
public Uri Endpoint { get; set; }
```

Reply:
But we're no longer using connection string to initialize the client (since we want to highlight the constructor that takes in (endpoint, credential, options)).

Reply:
ah, ok then ! :D